src/components/a-form-input/_template.js

```diff
@@ -0,0 +1,24 @@
+import html from 'nanohtml';
+
+export default function ({
+  inputId,
+  type = 'text',
+  placeholder = '',
+  value = '',
+  name,
+  disabled = false,
+}) {
+  return html`<div class="a-form-input__wrapper">
+      <input
+          id="${inputId}"
+          class="a-form-input__input" 
+          type="radio"
```

Comment:
is it correct to have input with type="radio" + 2 lines below anything you pass as a parameter ? so when i use the webcomponent with type="email", it just transforms to type="radioemail"

Reply:
agree we have already a radio component